lib/util/git/config.ts

```diff
@@ -17,13 +17,16 @@ export function getNoVerify(): GitNoVerifyOption[] {
   return noVerify;
 }
 
-export function simpleGitConfig(): Partial<SimpleGitOptions> {
+export function simpleGitConfig(extraConfig: SimpleGitOptions['config'] = []): Partial<SimpleGitOptions> {
```

Comment:
Isn't it possible to pass token in the same way other platforms do?

Reply:
yes, revert

Reply:
the token can't be passed as an URL argument